packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -66,6 +66,39 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     return null;
   }
 
+  getSchemasStream() {
+    const filePathFactory = (fileIndex: number = 0) => {
```

Comment:
It would be nice to follow the same logic used in the other `get{}Stream` (using the helpers, etc...)

Reply:
You can ignore this comment. I've done the change in #https://github.com/strapi/strapi/pull/14766